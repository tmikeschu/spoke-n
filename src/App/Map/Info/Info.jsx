import React from 'react'
import '../../App.css'

const Info = () => (
  <article className="info">
    <p>Have an amiga in Antigua?  A tía in Tijuana? Couch to crash on in Colombia? Bike shop in Bolivia?</p>
    <p><span>Let</span> <span>us</span> <span>know</span>!</p>
    <p>↓</p>
  </article>
)

export default Info

