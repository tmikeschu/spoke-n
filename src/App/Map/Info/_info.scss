.info {
  background: $yellow;
  color: #fff;
  font-size: 1.1em;
  padding: 5px;
  text-align: left;
  p {
    &:nth-of-type(2) {
      font-size: 1.5em;
      text-align: center;
      margin: 10px 0;
      span {
        text-decoration: underline;
      }
    }
    &:nth-of-type(3) {
      text-align: center;
    }
  }
  @media (max-width: 640px) {
    padding: 1px;
    line-height: 1.5em;
    font-size: .5em;
    text-align: left;
    p {
      &:nth-of-type(2) {
        text-align: center;
        margin: 10px 0;
        span {
          font-size: 1em;
          text-decoration: underline;
        }
      }
      &:nth-of-type(3) {
        text-align: center;
      }
      &:nth-of-type(4) {
        font-size: 8px;
      }
    }
  }
}
