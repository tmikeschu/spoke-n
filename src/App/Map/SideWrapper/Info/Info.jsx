import React from 'react'

const Info = () => (
  <article className="Info">
    <p>Have an amiga in Antigua?  A tía in Tijuana? Couch to crash on in Colombia? Bike shop in Bolivia?</p>
    <h4><span>Let</span> <span>us</span> <span>know</span>!</h4>
    <p>↓</p>
  </article>
)

export default Info
