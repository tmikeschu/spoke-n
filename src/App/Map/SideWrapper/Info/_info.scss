.Info {
  border-top: 4px $color1 solid;
  margin: 8px 0;

  h4 {
    text-align: center;
    margin: 8px 0 !important;
    span {
      text-decoration: underline;
    }
  }

  p {
    &:nth-of-type(2) {
      text-align: center;
    }
  }
}
