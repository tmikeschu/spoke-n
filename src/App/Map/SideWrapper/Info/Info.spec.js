import React from "react"
import { shallow } from "enzyme"
import Info from "./Info"

describe("<Info />", () => {
  const info = shallow(<Info />)

  it("renders without crashing", () => {
    expect(info).toBeTruthy()
  })

  it("renders content", () => {
    expect(info.equals(
      <article className="info">
        <p>Have an amiga in Antigua?  A tía in Tijuana? Couch to crash on in Colombia? Bike shop in Bolivia?</p>
        <p><span>Let</span> <span>us</span> <span>know</span>!</p>
        <p>↓</p>
      </article>
    )).toBeTruthy()
  })
})
