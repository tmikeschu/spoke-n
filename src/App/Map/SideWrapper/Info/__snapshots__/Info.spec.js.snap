// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Info /> renders content 1`] = `
<article
  className="Info"
>
  <p>
    Have an amiga in Antigua?  A tía in Tijuana? Couch to crash on in Colombia? Bike shop in Bolivia?
  </p>
  <h4>
    <span>
      Let
    </span>
     
    <span>
      us
    </span>
     
    <span>
      know
    </span>
    !
  </h4>
  <p>
    ↓
  </p>
</article>
`;
